src/utils/pst.ts

```diff
@@ -0,0 +1,80 @@
+import axios from "axios";
+
+// ArDrive Profit Sharing Community Smart Contract
+const vertoCacheURL =
+  "https://v2.cache.verto.exchange/-8A6RexFkpfWwuyVO98wzSFZh0d6VJuI-buTJvlwOJQ";
+
+// Calls the ArDrive Community Smart Contract to pull the fee
+export async function getArDriveFee(): Promise<number> {
```

Comment:
This function name implies that you're returning the whole fee as opposed to a percentage. Consider renaming to `getArDriveTipPercentage`.

Reply:
[Updated](https://github.com/th8ta/ArConnect/pull/33/commits/f76faa99ec9bfb48f340cbe9b4857e805e4e2582)